ports/libsimpleservo/capi/src/lib.rs

```diff
@@ -212,13 +218,22 @@ pub extern "C" fn servo_version() -> *const c_char {
 }
 
 #[cfg(target_os = "windows")]
-fn init_logger() {
+fn init_logger(_modules: &[*const c_char]) {
```

Comment:
Let's remove the `_` prefix, which is only for unused variables.

Reply:
Implemented.